test/test_correlation_sampling.jl

```diff
@@ -150,7 +150,7 @@ end
 
     sc = dynamical_correlations(sys; nω=10, ωmax=5.5, dt=2dt_langevin)
     add_sample!(sc, sys)
-    qs = [[0.0, 0.0, 0.0], [0.2, -0.4, 0.1]]
+    qs = [[0.0, 0.0, 0.0], [-0.2, 0.4, -0.1]]
```

Comment:
Sam figured it out. The sequence goes like this:

* Sunny release 0.5.11 was accidentally consistent because of cancellation between two bugs.
* After merging the fix to the spin dynamics in https://github.com/SunnySuite/Sunny.jl/pull/247, the result became _inconsistent_ because there was nothing to cancel the bug in the sign of Q.
* In this branch, the final bug in the sign of Q is fixed again (for `SampledCorrelations`) so the result is again consistent.